### ISSUE TITLE
Affinity indention error in Ocotpus server chart

### ISSUE THREAD

**Issue by dmitrii-didenko** [OctopusDeploy/helm-charts#244]
Hello!

There is an issue with affinity block in octopus server chart. Testing with the following values.yaml:
```
affinity:
  podAntiAffinity:
    requiredDuringSchedulingIgnoredDuringExecution:
      - labelSelector:
          matchExpressions:
            - key: app.kubernetes.io/component
              operator: In
              values:
                - ingester
        topologyKey: kubernetes.io/hostname
```
This throws an error: "error converting YAML to JSON: yaml: line 26: did not find expected key"

This is because of indention and combination of "{{" and `indent` instead of `nindent` in the statefulset template

My suggestion is to fix this the way it was fixed for tolerations, just by moving block to the left:
```
      {{- with .Values.affinity }}
      affinity:
{{ toYaml . | indent 8 }}
      {{- end }}
    {{- with .Values.octopus.tolerations }}
```
then, it shuld work, I've tested.

**Comment by liam-mackie** [comment]
Hi @dmitrii-didenko! Thanks for the report - we're working on an updated version of this helm chart and I've incorporated this issues resolution. Once we've published v1.0.0 of the chart, I'll let you know so you can test?

**Comment by APErebus** [comment]
@liam-mackie Can this issue be closed now?

**Comment by liam-mackie** [comment]
Yep! This has been fixed in v1.0.0 of the helm chart @dmitrii-didenko :)
